### ISSUE TITLE
Readd Contact Form

### ISSUE THREAD

**Issue by danry25** [AccelerateNetworks/NumberSearch#182]
**How will this improve revenue or reduce our costs? Please provide a complete explanation!**
@andrewstiefel has requested we re-add this Contact form as we rework the website. He plans to tweak the form to make it more appealing to potential clients.

**Describe the solution you'd like**
Revert the removal of this [data access model](https://github.com/AccelerateNetworks/NumberSearch/commits/master/NumberSearch.DataAccess/Models/ContactForm.cs) and this [HTML page](https://github.com/AccelerateNetworks/NumberSearch/commits/d83521b9326c4f949aecac100f37058dfd66775d/NumberSearch.Mvc/Views/Contact/Index.cshtml). 

**Additional context**
Expect a pull request from @andrewstiefel to change how this page is worded and formatted

**Comment by uncheckederror** [comment]
This page existed for over a year before we removed it. It had a large call to action button on the home page. Not a single customer submitted a contact request in that time. Customers were more likely to directly place an order for products on our website then they were to fill out a contact request form. The customers that do contact us through the website almost always contact sales directly by calling the phone numbers that we list on every page.

It seems unlikely to me that reviving and updating the content on the Contact Us page is going to produce any additional sales.

Do you genuinely believe that we are missing out on customer conversions because we don't have a contact us page?

**Comment by danry25** [comment]
@uncheckederror I hear you, hopefully @andrewstiefel can justify re-adding this form. I tried to find it on archive.org but it appears this form was both added and removed before the site was snapshotted again in 2020.

@andrewstiefel The [Talk To Us](https://web.archive.org/web/20201111193715/https://acceleratenetworks.com/) button used to link to this Contact Us form.

**Comment by andrewstiefel** [comment]
@uncheckederror and @danry25 It doesn't surprise me that no one filled out the form previously.

The goal is to create a sales lead generation page, not a generic contact us page, to drive revenue by emphasizing and expanding your sales and service story here. I think it is worth reworking this page with information about how to request a demo/quote, what to expect when working with you, and provide contact info (phone + email). I might even feature Scott and Kyle on the page with testimonials about working with them to build trust that you're talking to a real person.

Two examples from your competitors:
https://www.goto.com/connect/try
https://www.ringcentral.com/lp/small-business-phone-service.html

I agree that it is lower priority compared to other aspects I'm working on right now, but should be part of optimizing overall conversions by providing an alternative channel to the checkout flow and chat.

**Comment by uncheckederror** [comment]
@andrewstiefel ok, this first example is more or less what the original contact us page was before we gave up on it: https://www.goto.com/connect/try

This second example appears to be a contact form + sales pitch: https://www.ringcentral.com/lp/small-business-phone-service.html#ring-s-off

Both the current home page: https://acceleratenetworks.com/ and the about us page: https://acceleratenetworks.com/Home/About also fall into that role as they offer our direct contact info and a "why choose us" sales pitch.

We don't really have a testimonials page although the idea has come up before. Most of the customer feedback ends up on our Google reviews page: https://www.google.com/search?q=accelerate+networks which is nice because we don't maintain it and customers trust Google more than they trust our marketing materials.

The end goal is to get the customer to call or email us so that Scott and Kyle can do their sales magic, set the customers expectations, and build a relationship.

Currently to get a quote, you can fill out your cart with the things you want, and then check the "Quote" box on the checkout page. Typically this happens after the customer has talked to the sales team and the sales team creates this quote for them or walks them through the process of creating it themselves.

There's no formal process for requesting a demo at this time, rather this is something that is left to the sales team to figure out based on their understanding of the customer. 

Ideally we would keep the customer on our website for as short a time as possible. We want to funnel them into direct contract with the sales team as quickly as we can so they can take control.

**Comment by andrewstiefel** [comment]
I agree: the goal is to get customers to call or email so Scott and Kyle
can work with them.

As you pointed out, there is no clearly labeled path to engage the sales
team to request more information, a quote, or a demo. The only way to get a
quote is through the checkout cart, but there is no indication on the
website that you should follow that path. Having contact info visible is
important, but we also need to guide users to the actions we want them to
take.

I'm re-working the value proposition for the home page, so the sales page
will become a supplemental part of the customer experience detailing how
the team can help you, not the overall sales pitch. The page will include
and link back to Google testimonials about how Scott & Keith have helped
other business owners. I agree that you don't need a testimonials page, but
I recommend including testimonials within your marketing and sales pages.

You can view the proposed copy for the page in this document
<https://www.dropbox.com/s/93m114teq3h8w75/Accelerate-Networks_Content_2021%2004%2019.docx?dl=0>
(page 10) -- this is still a work in progress, not the final version. But
hopefully it gives you a better idea of what I'm imagining.

Is there concern about development and maintenance of the form? We could
drop that from the page if that makes upkeep easier.

Please let me know how you would like to proceed. I agree that there are
other more urgent updates (like the home page) that need attention first.
